Frontend/src/app/model/restaurant.model.ts

```diff
@@ -8,6 +9,8 @@ export type Restaurant = {
 	phoneNumber: string;
 	logoName: string;
 	logo: string | null;
```

Comment:
Can't you use undefined instead of null? That way you could just do `logo?: string`